torch/_subclasses/fake_tensor.py

```diff
@@ -783,11 +790,9 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             ), f"{args} {kwargs}"
             return converter(self, args[0])
 
-        if self.check_for_non_fake(flat_arg_tensors):
-            raise Exception(
-                "Invoking operators with non-Fake Tensor inputs in FakeTensorMode is not yet supported. "
-                f"Please convert all Tensors to FakeTensors first. Found in {func}(*{args}, **{kwargs})"
-            )
+        args, kwargs = self.validate_and_convert_non_fake_tensors(
```

Comment:
could we either `del flat_arg_tensors` after the call here or update it to be correct ? if someone were to add another use of `flat_arg_tensors` later it would be outdated